interfaces/builtin/browser_support.go

```diff
@@ -230,6 +230,9 @@ owner /{dev,run}/shm/WK2SharedMemory.* mrw,
 
 # Chromium content api on (at least) later versions of Ubuntu just use this
 owner /{dev,run}/shm/shmfd-* mrw,
+
+# Allow monitoring process memory utilization (used by chromium)
+owner @{PROC}/@{pid}/clear_refs w,
```

Comment:
This looks fine since while this would allow the snap to 'w'rite to the file for any pid with matching owner (ie, it is not specific to the snap), it is in browserSupportConnectedPlugAppArmorWithSandbox which requires a snap declaration to use.